### PR TITLE
MedicalTorch integration: transforms

### PR DESCRIPTION
**Pull request by charleygros**
### Done:
- `ToTensor`
- `ToPIL`
- `IMEDTransform`
- `UndoTransform`
- `StackTensors`
- `Crop`
- `CenterCrop2D`
- `Normalize`
- `NormalizeInstance`
- `NormalizeInstance3D`
- `RandomRotation`
- `RandomRotation3D`
- `RandomFlip3D`
- `RandomAffine`
- `RandomTensorChannelShift`
- `ElasticTransform`
- `Resample`
- `AdditiveGaussianNoise`
- `Clahe`
- `Clipping`
- Correct imports across `ivadomed`
- Run tests

### CODE REVIEW

**Comment by charleygros** [comment]
I am pulling `Normalize()` (from [here](https://github.com/neuropoly/medicaltorch/blob/master/medicaltorch/transforms.py#L293)) even if we are not using it at the moment in `ivadomed` since it could be of interest at some point.

File: ivadomed/transforms.py [hunk 1]
```diff
@@ -211,30 +457,30 @@ def __call__(self, sample):
...
-                gt_metadata[i]["__centercrop"] = (fh, fw, h, w)
             rdict['gt'] = gt_data
-            rdict['gt_metadata'] = gt_metadata
 
         sample.update(rdict)
         return sample
 
 
-class DilateGT(mt_transforms.MTTransform):
+class DilateGT(IMEDTTransform):
```
Comment (olix86) [inline ivadomed/transforms.py:483]:
```suggestion
class DilateGT(IMEDTransform):
```

**Comment by jcohenadad** [comment]
As discussed in meeting, we could split into multiple files, each file dealing with a task (normalization, cropping, etc.)

Also: it would be important to keep all the possible transforms available from the package: when writing code, having the auto-completion showing all the transfo.

**Comment by charleygros** [comment]
```
from torchvision import transforms
```
to
```
from torchvision import torch_transforms
```

File: testing/test_inference.py [hunk 2]
```diff
@@ -80,12 +80,8 @@ def test_inference(film_bool=False):
                              collate_fn=imed_loader_utils.imed_collate,
                              num_workers=1)
 
-    if film_bool:
```
Comment (charleygros) [inline testing/test_inference.py:83]: Not maintained anymore.

File: testing/test_adaptative.py [hunk 3]
```diff
@@ -3,9 +3,9 @@
 import torch
 from medicaltorch import transforms as mt_transforms
```
Comment (olix86) [inline testing/test_adaptative.py:4]:
```suggestion

```

**Comment by olix86** [comment]
Great work! I think we can safetly remove mt from `requirements.txt`?

File: ivadomed/transforms.py [hunk 4]
```diff
@@ -1,23 +1,80 @@
 import math
+import numbers
 import random
 import numpy as np
 from PIL import Image
-import torch
-import torchvision.transforms.functional as F
-from scipy.ndimage.measurements import label, center_of_mass
 
+from skimage.exposure import equalize_adapthist
```
Comment (olix86) [inline ivadomed/transforms.py:7]: I've added it to reqs, was missing
